users/urls.py

```diff
@@ -1,10 +1,14 @@
 from django.conf.urls import url
 
-from .views import UserLoginView, DashboardView, UserLogoutView, RegisterView
+from .views import (UserLoginView, DashboardView, UserLogoutView, RegisterView,
+                    UserProfileView, UpdateProfileView, UpdatePasswordView)
 
 urlpatterns = [
     url(r'^login/$', UserLoginView.as_view(), name='user_login'),
     url(r'^dashboard/$', DashboardView.as_view(), name='dashboard'),
     url(r'^logout/$', UserLogoutView.as_view(), name="user_logout"),
     url(r'^registration/$', RegisterView.as_view(), name='register'),
+    url(r'^profile/$', UserProfileView.as_view(), name="user_profile"),
```

Comment:
use single quote. someone already started it so just follow his format unless his are incorrect